docs/mast/mast.rst

```diff
@@ -868,6 +868,97 @@ To access sector information at a particular location there is  `~astroquery.mas
                 tess-s0010-1-4     10      1   4
 
 
+Zcut
+====
+
+Zcut for MAST allows users to request cutouts from various Hubble deep field surveys. The cutouts are
+returned as full-frame image (FFI) cutouts. This tool can be accessed in Astroquery by using the Zcut
+class.
+
+
+Cutouts
+-------
+
+The ~astroquery.mast.ZcutClass.get_cutouts function takes a coordinate and cutout size (in pixels or
+an angular quantity) and returns the cutout target pixel file(s) as a list of ~astropy.io.fits.HDUList
+objects.
+
+If the given coordinate appears in more than one Zcut survey, a target pixel file will be produced for
+each survey. If the cutout area overlaps more than one camera or ccd a target pixel file will be produced
+for each one.
+
+.. code-block:: python
+
+                >>> from astroquery.mast import Zcut
+                >>> from astropy.coordinates import SkyCoord
+
+                >>> cutout_coord = SkyCoord(189.49206, 62.20615, unit="deg")
+                >>> hdulist = Zcut.get_cutouts(coordinates=cutout_coord, size=5)
+                >>> hdulist[0].info()
+                Filename: <class '_io.BytesIO'>
+                No.    Name      Ver    Type      Cards   Dimensions   Format
+                  0  PRIMARY       1 PrimaryHDU      11   ()
+                  1  CUTOUT        1 ImageHDU       177   (5, 5)   float32
+                  2  CUTOUT        1 ImageHDU       177   (5, 5)   float32
+                  3  CUTOUT        1 ImageHDU       177   (5, 5)   float32
+
+
+The ~astroquery.mast.ZcutClass.download_cutouts function takes a coordinate and cutout size (in pixels or
+an angular quantity) and downloads the cutout target pixel file(s) as either fits files or image (png/jpg)
+files.
+
+If a given coordinate appears in more than one Zcut survey, a target pixel file will be produced for each
+survey. If the cutout area overlaps more than one camera or ccd a target pixel file will be produced for each
+one.
```

Comment:
Delete "If the cutout area overlaps more than one camera or ccd a target pixel file will be produced for each
one." (TESS specific)